curriculum/challenges/english/25-front-end-development/quiz-dom-manipulation-and-click-event-with-javascript/66edd07682767adff3a6231e.md

```diff
@@ -17,439 +17,560 @@ Answer all of the questions below correctly to pass the quiz.
+```js
+document.body.append("<hr/>");
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following statements is false?
 
 #### --distractors--
 
-Placeholder distractor 1
+`window` is an alias for `navigator.window
+
+```js
+window === navigator.window // returns true
+```
 
 ---
 
-Placeholder distractor 2
+You can use the user screen properties (like position and size) using the `Window` object.
 
 ---
 
-Placeholder distractor 3
+`navigation` is an alias for `window.navigation`
+
+```js
+navigation === window.navigation // returns true
+```
 
 #### --answer--
 
-Placeholder answer
+You can access the clipboard using :
+
+```js
+await browser.clipboard.read();
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Assuming the JavaScript variable `element` is an HTML `Element` object, which code does not do the same thing as the others?
 
 #### --distractors--
 
-Placeholder distractor 1
+```js
+element.toggleAttribute("disabled");
+```
 
 ---
 
-Placeholder distractor 2
+```js
+element.hasAttribute("disabled") ? element.removeAttribute("disabled") : element.setAttribute("disabled", "");
+```
 
 ---
 
-Placeholder distractor 3
+```js
+element.toggleAttribute("disabled", !element.hasAttribute("disabled"));
+```
 
 #### --answer--
 
-Placeholder answer
+```js
+element.setAttribute("disabled", !element.getAttribute("disabled"));
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of these does not set the element text color to `red`.
 
 #### --distractors--
 
-Placeholder distractor 1
+```html
+<p style="color: red;">...</p>
+```
 
 ---
 
-Placeholder distractor 2
+```html
+<p id="paragraph">...</p>
+<script>document.getElementById("paragraph").style.color = "red";</script>
+```
 
 ---
 
-Placeholder distractor 3
+```html
+<style>.red-txt { color: red; }</style>
+<p id="paragraph">...</p>
+<script>document.getElementById("paragraph").classList.add("red-txt");</script>
+```
 
 #### --answer--
 
-Placeholder answer
+```html
+<p color="red">...</p>
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+In the case of a click event, which `Event` property returns the clicked sub-element that triggerred the parent's callback?
 
 #### --distractors--
 
-Placeholder distractor 1
+`element`
 
 ---
 
-Placeholder distractor 2
+`currentTarget`
 
 ---
 
-Placeholder distractor 3
+`type`
 
 #### --answer--
 
-Placeholder answer
+`target`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which `Event` property returns the HTML element that the event listener was attached to?
 
 #### --distractors--
 
-Placeholder distractor 1
+`target`
 
 ---
 
-Placeholder distractor 2
+`element`
 
 ---
 
-Placeholder distractor 3
+`type`
 
 #### --answer--
 
-Placeholder answer
+`currentTarget`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which `Event` function can be called to prevent an even from bubbling up, without denying the others element's events?
```

Comment:
```suggestion
Which `Event` function can be called to prevent an event from bubbling up without affecting other listeners on the current target?
```